app/assets/stylesheets/application.scss

```diff
@@ -1,44 +1,68 @@
 @import "bootstrap";
 
+$orange: #f2580b;
+@mixin orange_background {
```

Comment:
@mixin declaration should be preceded by an empty line<br>Name of mixin `orange_background` should be written in all lowercase letters with hyphens instead of underscores